### DISCUSSION TITLE
Local state

### DISCUSSION THREAD

**Discussion by danielvanmil** [apollo-elements/apollo-elements#281] [Ideas]
Hello,

https://apolloelements.dev/guides/usage/local-state/ describes handling local state.
Apollo (React) client has an useReactiveVar API (https://www.apollographql.com/docs/react/local-state/reactive-variables/#usereactivevar-hook). Does Apollo elements also provide an API to react on var changes in for example lit element without using a query or is this planned in the future?

Thanks.

**Comment by bennypowers** [comment]
https://github.com/apollo-elements/apollo-elements/blob/main/packages/core/reactive-variable-controller.ts
